boxoffice/templates/layout.html.jinja2

```diff
@@ -35,3 +37,12 @@
   </div>
 {%- endblock %}
 {%- endblock %}
+
+{% block jquery -%}
+<script src="//ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
```

Comment:
We should update jQuery globally, in Baseframe. No good reason to use an older version anywhere.